services/livesync-service-base.ts

```diff
@@ -7,6 +7,7 @@ import * as shell from "shelljs";
 import * as path from "path";
 import * as temp from "temp";
 import * as minimatch from "minimatch";
+import * as constants from "../../constants";
```

Comment:
because constants like TNS_IOS_RUNTIME_NAME, TNS_ANDROID_RUNTIME_NAME, etc. are not valuable for the common lib. They are usable only for NativeScript CLI

https://github.com/NativeScript/nativescript-cli/blob/master/lib/constants.ts#L12